migrator.go

```diff
@@ -175,6 +198,24 @@ func (m Migrator) DropTable(values ...interface{}) error {
 	return nil
 }
 
+func (m Migrator) AddColumn(value interface{}, field string) error {
+	if err := m.Migrator.AddColumn(value, field); err != nil {
+		return err
+	}
+	return m.RunWithValue(value, func(stmt *gorm.Statement) error {
+		if field := stmt.Schema.LookUpField(field); field != nil {
+			if field.Comment != "" {
+				if err := m.DB.Exec(
+					"COMMENT ON COLUMN " + stmt.Table + "." + field.DBName + " is " + field.Comment,
```

Comment:
Hello @kainonly 

I found two issues in this line of code, which also exists on following code: 

1, `field.Comment` should be used as param
2, `COMMENT ON COLUMN ` should be written like "COMMENT ON COLUMN ?" with params `m.CurrentTable(stmt)`?